bin/macos/makefile

```diff
@@ -53,8 +62,19 @@ SDL2_rotozoom.o: SDL2_rotozoom.c SDL2_rotozoom.h SDL2_gfxPrimitives.h SDL_stbima
 flood.o: flood.c
 	$(CXX) -c -O3 $< -o $@
 
-bbdata.o: ../../src/bbdata_x86_64.nas
-	nasm --prefix _ -f macho64 -s ../../src/bbdata_x86_64.nas -o bbdata.o
+sort.o: sort.c
+	$(CXX) -c -O3 $< -o $@
+
+ifeq ($(ARCH),arm64)
+    BBDATA_COMPILE = clang -mmacosx-version-min=$(MIN_MACOS_VERSION) -c
+    BBDATA_SRC = ../../src/bbdata_arm_64.s
+else 
+    BBDATA_COMPILE = nasm --prefix _ -f macho64 -s
+    BBDATA_SRC = ../../src/bbdata_x86_64.nas
```

Comment:
Compile `bbdata_x86_64.nas` with `nasm` for x86 (same as before), and `bbdata_arm_64.s` with `clang` for ARM.